pkg/operator/ceph/csi/spec.go

```diff
@@ -498,6 +522,23 @@ func deleteCSIDriverResources(
 	return succeeded
 }
 
+func applyCephClusterNetworkConfig(objectMeta *metav1.ObjectMeta, rookclientset rookclient.Interface) (bool, error) {
+	cephClusters, err := rookclientset.CephV1().CephClusters(objectMeta.Namespace).List(metav1.ListOptions{})
+	if err != nil {
+		return false, errors.New(fmt.Sprintf("failed to find CephClusters in namespace %q", objectMeta.Namespace))
```

Comment:
Use `errors.Errorf()` instead of `fmt.Sprintf()` inside `errors.New()`.

Reply:
`errors.Errorf()` returns error so it can' t be used inside `errors.New()`
Using `errors.Errorf()` instead of `errors.New()`

Reply:
Maybe I wasn't clear, just use `errors.Errorf()`

Reply:
done